Cannot find frozen with `find_package(frozen REQUIRED)`

I installed [frozen from Fedora repository](https://packages.fedoraproject.org/pkgs/frozen/frozen-devel/)

```
CMake Error at CMakeLists.txt:12 (find_package):
  Could not find a configuration file for package "frozen" that is compatible
  with requested version "1.1.0".

  The following configuration files were considered but not accepted:

    /usr/share/cmake/frozen/frozenConfig.cmake, version: 1.1.0 (32bit)
```

Maybe the problem is similar to the following issue, see problem two in https://github.com/nemtrif/utfcpp/issues/104

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,9 +126,13 @@ if(frozen.installation)
   # generate a CMake configuration file for consumption by CMake's `find_package`
   # intrinsic
   #
+  if(${CMAKE_VERSION} VERSION_GREATER "3.14")
+    set(OPTIONAL_ARCH_INDEPENDENT "ARCH_INDEPENDENT")
+  endif()
   write_basic_package_version_file("frozenConfigVersion.cmake"
     VERSION ${PROJECT_VERSION}
-    COMPATIBILITY SameMajorVersion)
+    COMPATIBILITY SameMajorVersion
+    ${OPTIONAL_ARCH_INDEPENDENT})
 
   install(FILES "${PROJECT_BINARY_DIR}/frozenConfigVersion.cmake"
     DESTINATION "${CMAKE_INSTALL_DATADIR}/cmake/frozen")

